react-ui/src/App.js

```diff
@@ -28,7 +28,7 @@ class App extends Component {
         <Route path="/login" component={LogIn}>
           {/* <LogIn /> */}
         </Route>
-        <Route exact path="/dashboard/:email/:type/:name" component={Dashboard}>
+        <Route exact path="/dashboard/:email/:userType/:firstName/:personId/:accessToken" component={Dashboard}>
```

Comment:
Please note that additional props were needed from the login page. Although this method is not a secure way of passing sensitive parameters such as accessToken, adjustments such as implementing a redux store or leveraging another method is out of scope for this ticket and will be considered for a future sprint.

Reply:
This looks good, thank you for the reasoning.